config/job.js

```diff
@@ -60,13 +60,26 @@ const SCHEMA_STEP_OBJECT = Joi.object()
 const SCHEMA_STEP = Joi.alternatives().try(SCHEMA_STEP_STRING, SCHEMA_STEP_OBJECT);
 const SCHEMA_STEPS = Joi.array().items(SCHEMA_STEP).min(1);
 const SCHEMA_IMAGE = Joi.string();
+const SCHEMA_EMAIL = Joi.string().email();
+const SCHEMA_EMAILS = Joi.array()
+    .items(SCHEMA_EMAIL)
+    .min(0);
+const SCHEMA_NOTIFICATIONS = Joi.object()
```

Comment:
Why does it need to be `notifications_*`? Why not just `email`, `hipchat`, `slack`, `featureA`?